index.html

```diff
@@ -8,29 +8,173 @@
+    </div>
+    <div >
+        <ul class="program1">
+            <li class="Program-skilhtml">html</li>
+            <li class="Program-skilhtml">css</li>
+            <li class="skiljava">javascript</li>
+        </ul>
+    </div>
+        
+    <div class="buttonsee">
+        <button class="button1"> See Project</button>
+    </div>
+     </div>
+  </section>
+  <section class="Workssection">
+    <div class="cardBack">
+    <a class="card4" href="#"></a>
+    
+        <h3  class="tonic" >Multi-Post
+            Stories </h3>
+        <div>
+    <ul class="works">
+        <li class="canopy"> CANOPY</li>
+        <li class="BackendDev">BackendDev</li>
+        <li class="num"> 2015 </li>
+     </ul>
+    <p class="section1"> A daily selection of privately personalized reads; no accounts or sign-ups required </p>
+</div>
+<div >
+    <ul class="program1">
+        <li class="Program-skilhtml">html</li>
+        <li class="Program-skilhtml">css</li>
+        <li class="skiljava">javascript</li>
+    </ul>
+</div>
+   <div class="buttonsee">
+    <button class="button1"> See Project</button>
+</div>
+</div>
+  </section>
+  <section class="Workssection">
+    <div class="cardBack2">
+        <h3  class="tonic" >About Myself </h3>
+        <div>
+    <ul class="works">
+        <li class="canopy"> CANOPY</li>
+        <li class="BackendDev">BackendDev</li>
+        <li class="num"> 2015 </li>
+     </ul>
+    <p class="section1"> Hello I’m a software developer! I can help you build a product , 
+        feature or website Look through some of my work and experience! If you like what you see 
+        and have a project you need coded, don’t hestiate to contact me. </p>
+</div>
+<div class = "social-section">
+    <h4 class="connectmywork"> LET’S CONNECT</h4>
+    <ul class="social-icons-mywork">
+        <li><a href="#"><img src="./image/Vectortweet.png" alt=""></a></li>
+        <li><a href="#"><img src="./image/VectorM.jpg" alt=""></a></li>
+        <li><a href="#"><img src="./image/Vector5.png" alt=""></a></li>
+        <li><a href="#"><img src="./image/VectorGit.png" alt=""></a></li>
+        <li><a href="#"><img src="./image/Linkedin.svg" alt=""></a></li>
+    </ul>
 
+</div>
+<div class="buttonsee">
+    <button class="button1"> Get my resume </button>
+</div>
+        <ul class="Program-skil">
+            <li  class="Programs-lang" href="#"><img  src="./image/Frame 3.png" alt=""></li>
+            <ul class="Program-skil">
+                <li  class="Programs" href="#"><img src="./image/Frame 9.png" alt=""></li>
+                  <li  class="Programs" href="#"><img src="./image/Frame 10.png" alt=""></li>
+                  <li  class="Programs" href="#"><img src="./image/Frame 11.png" alt=""></li>
+              </ul>
+            <li  class="Programs-lang" href="#"><img src="./image/Frame 01.png" alt=""></li>
+            <li  class="Programs-lang" href="#"><img src="./image/Frame 02.png" alt=""></li>
+        </ul>
```

Comment:
- [Optional] Iam sorry that it was not pointed out by the previous reviewer, I strongly suggest you to change the languages from image to text as it is not an image. they are texts.
![img](https://user-images.githubusercontent.com/66992901/199460867-3a78b762-5e56-47e9-97a0-47be2b7dd7c3.PNG)